Init sig-device-iot info (#21)
kubeedge/community: sig-device-iot/README.md

# SIG Device IoT

SIG Device IoT is responsible to simplify, develop, and maintain device (protocols) integrations as extensions, or add-ons, to KubeEdge deployments.

> The [charter](./charter.md) defines the scope and governance of the Auth Special Interest Group.

Reviewer: ```suggestion
The [charter](./charter.md) defines the scope and governance of the Device IoT Special Interest Group.
```

## Meetings

> * Regular SIG Meeting: [Friday at 10:00 UTC+8](https://zoom.us/j/4167237304) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:00&tz=PT%20%28Pacific%20Time%29).

Reviewer: ```suggestion
* Regular SIG Meeting: [Friday at 10:00 UTC+8](https://zoom.us/j/4167237304) (biweekly, starts on Sep. 25th 2020). [Convert to your timezone](https://www.thetimezoneconverter.com/?t=10%3A00%20am&tz=GMT%2B8&).
```

 * [Meeting notes and Agenda](https://docs.google.com/document/d/1PeuZzfLjU8oIduKI2nFeckKAhhckME4tqCZ33WmORBI/edit#heading=h.252sw1n3b1vb).
 * [Meeting Calendar](https://calendar.google.com/calendar/embed?src=8rjk8o516vfte21qibvlae3lj4%40group.calendar.google.com) | [Subscribe](https://calendar.google.com/calendar?cid=OHJqazhvNTE2dmZ0ZTIxcWlidmxhZTNsajRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)
 
## Contact

> - Slack: [#sig-node](https://kubeedge.slack.com/archives/C01239D6PM4)

Reviewer: ```suggestion
- Slack: [#sig-device-iot](https://kubeedge.slack.com/archives/C01239D6PM4)
```
